frontend/src/middleware.ts

```diff
@@ -13,6 +13,20 @@ function isExcluded(pathname: string) {
   return excludedPaths.some((excludedPath) => pathname.includes(excludedPath))
 }
 
+function isRootAssetForward(pathname: string) {
+  const forwardedPaths = [
+    '/apple-touch-icon-120x120-precomposed.png',
+    '/apple-touch-icon-120x120.png',
+    '/apple-touch-icon-152x152-precomposed.png',
+    '/apple-touch-icon-152x152.png',
+    '/apple-touch-icon-precomposed.png',
+    '/apple-touch-icon.png',
+    '/favicon.ico'
+  ]
```

Comment:
I feel like this might be too restrictive.

https://developer.apple.com/design/human-interface-guidelines/app-icons#Specifications